src/sfloader/fluid_sffile.c

```diff
@@ -649,11 +662,15 @@ static int process_info(SFData *sf, int size)
         uint32_t *fcc;
     } item;
     unsigned short ver;
+    unsigned char *p;
+    sf->default_mod_list = new_fluid_list();
```

Comment:
```suggestion
    sf->default_mod_list = NULL;
```
Otherwise there would an empty list element at the end of the list.